centos/centos-7.x/common/install_dcgm.sh

```diff
@@ -5,7 +5,7 @@ set -ex
 DCGM_VERSION=2.3.1
 $COMMON_DIR/write_component_version.sh "DCGM" ${DCGM_VERSION}
 DCGM_URL=https://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/datacenter-gpu-manager-${DCGM_VERSION}-1-x86_64.rpm
-$COMMON_DIR/download_and_verify.sh $DCGM_URL "4bf0cccc4764f59b69e1df3b104ba8ecb77dc4bd98264b89d20aabd290808100"
+$COMMON_DIR/download_and_verify.sh $DCGM_URL "586bf03a7b0c9827c80dc0a82c6e8fe780ff1d76d82b103866906e4cdd191710"
```

Comment:
why do we need to change the checksum if the link is not changing?

Reply:
This is a change from nvidia's end.